### DISCUSSION TITLE
How to use nvim-tree instead?

### DISCUSSION THREAD

**Discussion by uyha** [LazyVim/LazyVim#352] [Q&A] [answered]
I want to swap out `neo-tree.nvim` and replace it with `nvim-tree.lua`. This does the trick 

```lua
return {
  { "nvim-neo-tree/neo-tree.nvim", enabled = false },
  {
    "nvim-tree/nvim-tree.lua",
    dependencies = { "nvim-tree/nvim-web-devicons" },
    config = function()
      require("nvim-tree").setup({
        actions = {
          open_file = {
            resize_window = false,
          },
        },
      })
    end,
    keys = {
      { "<C-n>", function() vim.cmd([[NvimTreeToggle]]) end, mode = { "n", "t" }, desc = "Toggle nvim-tree" },
      { "<C-f>", function() vim.cmd([[NvimTreeFindFile]]) end, desc = "Show current file in nvim-tree" },
    },
  },
}
```

However, the file tab doesn't take nvim-tree into consideration
![image](https://user-images.githubusercontent.com/8091245/222002573-415dcefe-d2bd-4ae7-906e-2090190a23c4.png)

How do I adjust the config to make this work?

**Comment by dpetka2001** [comment, marked as answer]
[This](https://www.lazyvim.org/plugins/ui#bufferlinenvim) is what makes the offset for neo-tree, so that bufferline takes it into consideration and cuts off just where neo-tree starts. You should try something similar with nvim-tree.
